index.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <!-- navbar -->
+      <nav class="navbar">
+
+        <div class="logo">
+          <img src="my_logo.png">
+        </div>
+
+        <div class="union">
+          <img src="union.png">
+        </div>
+
+      </nav>
+    <!-- introduction -->
+      <section class="introduction">
+        <h1 class="header">
+          I'm Aleem <br>
+          Glad to see <br>
+          you!
+        </h1>
+        <p class="intro">
+          I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <p>
+          LET’S CONNECT
+        </p>
```

Comment:
- [ ] This suppose `a` or `button` tag, and kindly follow the button styling: Active, Hover and Pressed as we have it in the design. 👍